extensions/wikia/VisitSource/scripts/VisitSource.js

```diff
@@ -0,0 +1,9 @@
+$(function() {
+	'use strict';
+
+	var sessionVisitSource = new VisitSource('WikiaSessionSource', window.wgCookieDomain),
+		lifetimeVisitSource = new VisitSource('WikiaLifetimeSource', window.wgCookieDomain, false);
```

Comment:
Does it make sense when you can quickly go to source file and see the same variable name?\
Additionally new object is stored in a variable with descriptive name ;)